src/components/Nav/StatusModal/StatusModal.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import * as S from './StatusModal.style';
+
+const StatusModal = ({
+  isOpenStatus,
+  handleStatusModal,
+  handleModal,
+  checkLogin,
+}) => {
+  return (
+    <S.StatusBackground
+      name="status"
+      hidden={!isOpenStatus}
```

Comment:
```suggestion
      visible={isOpenStatus}
```
부정으로 생각하는 것보다 긍정으로 생각하는게 간단하지 않을까요?